### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewSearch_ScreenReader: ScreenReader does not read the state of the "Toggle Replace" button as expanded/collapsed.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52268]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while JAWS is ON.
2. Navigate to Activity Bar and select "Search"(Cntrl+Shift+F) button.
3. Navigate to "Toggle Replace" button. Use "Enter/Space Bar" to expand/collapse.

**Actual:**
JAWS does not read the state of the "Toggle Replace" button as expanded/collapsed.

**Same issue has been observed with NVDA.**

**Expected:**
JAWS should read the state of the "Toggle Replace" button as expanded/collapsed.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader users will not know about the state of the button and its use(display/hide content).

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_ViewSearch_ScreenReader_ExpandCollapse.pptx](https://github.com/Microsoft/vscode/files/2114178/A11y_VSCode_ViewSearch_ScreenReader_ExpandCollapse.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
#52268
